### ISSUE TITLE
Rerun neuroblastoma crunch with smashing and gene conversion capability

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/refinebio#370]
### Context

As of #182, we did not have smashing in place and we'd like to test that as well as part of #230.

### Problem or idea

I would like the neuroblastoma datasets here run two ways: https://github.com/AlexsLemonade/refinebio/issues/112#issuecomment-363810177

* Aggregated by species, z-scored (StandardScaler)
* Aggregated by experiment, no transform

### Solution or next step

Depends on #307 

### New Issue Checklist

- [x] The title is short and descriptive.
- [x] You have explained the context that led you to write this issue.
- [x] You have reported a problem or idea.
- [x] You have proposed a solution or next step.

**Comment by kurtwheeler** [comment]
@jaclyn-taroni This dataset was rerun shortly after https://github.com/AlexsLemonade/refinebio/issues/335 finished. It is available via staging.refine.bio. The only thing that may not currently work for it is the conversion of Illumina Beadchip genes, which is dependent on https://github.com/AlexsLemonade/refinebio/pull/375.

Do you happen to know off the top of your head whether or not any of those experiments used Illumina Beadchip? If you don't I can go through the list and check. If the answer is no then this dataset should be good to go! If the answer is yes then I should rererun it.

**Comment by jaclyn-taroni** [comment]
I am poking around this morning in the interest of grabbing some data for training and `GSE16254` is a Affymetrix GEO SuperSeries that says it is submitter-processed? I checked the [GEO accession](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE16254) -- looks like raw data is available.

**Comment by jaclyn-taroni** [comment]
@kurtwheeler & @jaclyn-taroni will check this out

**Comment by jaclyn-taroni** [comment]
Closing as this has been superseded by #642
